### ISSUE TITLE
Isilon driver does not work with chown operations

### ISSUE THREAD

**Issue by clintonskitson** [emccode/rexray#253]
I was trying to run the postgres container and noticed that Isilon does not work when it comes to attempts to perform chown's within directories that are mounted.  Is this expected @danforrest

```
docker run -ti --volume-driver=rexray -v postgres:/var/lib/postgresql/data postgres
chown: changing ownership of ‘/var/lib/postgresql/data’: Operation not permitted
```

**Comment by clintonskitson** [comment]
```
/tt1 # adduser test
Changing password for test
New password: 
Bad password: similar to username
Retype password: 
Password for test changed by root
/tt1 # chown test:test test
chown: test: Operation not permitted
```

**Comment by danforrest** [comment]
Did you do this with the latest changes I added?  The original code had all volumes mounted as nobody:nobody.  The new code creates volumes as the user specified in the config.

**Comment by clintonskitson** [comment]
Give it a shot.  Used the latest commit and same results.

**Comment by clintonskitson** [comment]
@danforrest have you had a chance to check this one out?

**Comment by danforrest** [comment]
Working on it right now.  I'm trying to get docker/rexray setup with the latest branch to test it out.

**Comment by danforrest** [comment]
@clintonskitson Looks like the bug for this was in GoIsilon.  I've merged in the change so the latest GoIsilon code has the fix now.  Should I update the glide.yaml in the size_and_preemption PR or keep it separate?  I'll be submitting a PR for snapshots (hopefully today) that will include it by default it fhat'd be better.

**Comment by clintonskitson** [comment]
@danforrest Since it's just an update to the goisilon package, I would just
include it in the size_and_preemption PR.

**Comment by clintonskitson** [comment]
Fixed in 0.3.2-rc1 with proper privileges
